.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+## What does this PR address?
+<!-- Thanks for sending a pull request! -->
+
+<!-- Remove if not applicable -->
+Fixes # (issue)
+
+## Before submitting:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask -->
+<!--- If you plan to update documentation or tests in follow-up, please note -->
+
+- [ ] Make sure to follow [GitHub's guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) on creating PR.
+- [ ] Did you read through [contribution guidelines](https://github.com/railwayapp/nixpacks/blob/main/CONTRIBUTING.md)?
+- [ ] Can you build the project with `cargo build` and run the example with `cargo run --build examples/node --name node`?
+- [ ] Did your changes require updates to the documentation? Have you updated those accordingly?
+- [ ] Did you write tests to cover your changes? Did it pass locally when running `cargo test`?
+- [ ] Did you run `cargo check` and `cargo clippy` locally to ensure that CI pass?
+
+## Who can help review?
```

Comment:
I'm not so sure on this section. I think most contributors and maintainers just check over PRs, and then can just be forwarded on.

Reply:
IMO, I'd say remove it.

Reply:
sounds good.

Reply:
updated.